### ISSUE TITLE
scroll various target divs with one single scroll (not simultaneously)

### ISSUE THREAD

**Issue by bezin** [malihu/malihu-custom-scrollbar-plugin#213]
Hello @ all,

I am trying to build a website with a single scrollbar, but various scrolling targets. It is basically an almost fullscreen slideshow. The horizontal scrolling is done via JavaScript and must not be done by the user, therefore no horizontal scrollbar is necessary. Vertical scrolling, though, should be possible for the user by one single scrollbar at the browsers' right to fake browser native behaviour with a much nicer scrollbar ;) 

The script that takes care of horizontal scrolling also toggles active classes on the slides. The one in focus gets class="active".  The custom scrollbar should only scroll the slide vertically, that is set active by the script. 

TL;DR: One fixed single scrollbar at the browsers' right, scrolling various div's (not simultaneously), that varies during script run time.

I am aware that this is not a basic feature of this wonderful plugin and the plugin needs changes, which won't be update-proof. But maybe somebody out here as ever tried that?

Best, 
bezin
